packages/example/pages/_app.tsx

```diff
@@ -146,6 +147,9 @@ const sei = {
   contracts: {},
 } as const satisfies Chain;
 
+// Enable coinbase smart wallet feature
+coinbaseWallet.preference = 'smartWalletOnly';
+
```

Comment:
We'll want to enable the coinbase wallet option in the example dApp; we could test the types here by passing `all` though

Reply:
Yep. Done 👍